wishlist-backend/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
you don't need this line?

Reply:
Certainly dont need this one : # Create your models here.